assets/src/edit-story/components/panels/design/imageAccessibility/imageAccessibility.js

```diff
@@ -43,29 +46,47 @@ const MIN_MAX = {
   },
 };
 
+const HighlightRow = styled(Row)`
+  ${({ focusContainerSelector }) => focusContainerSelector} {
+    border-radius: 4px;
+  }
+  ${({ focusContainerCss }) => focusContainerCss}
+`;
+
 function ImageAccessibilityPanel({ selectedElements, pushUpdate }) {
   const resource = useCommonObjectValue(
     selectedElements,
     'resource',
     DEFAULT_RESOURCE
   );
   const alt = getCommonValue(selectedElements, 'alt', resource.alt);
+  const { highlight } = useHighlights(({ assistiveText }) => ({
+    highlight: assistiveText,
+  }));
+
+  const ref = useRef(null);
+
+  useEffect(() => {
+    highlight && ref.current?.focus();
+  });
```

Comment:
Should this run everytime `highlight` changes rather than every render?
```suggestion
  useEffect(() => {
    highlight && ref.current?.focus();
  }, [highlight]);
```

Reply:
This goes for the other ones too!